code/modules/admin/admin.dm

```diff
@@ -767,11 +767,7 @@
 		var/J_totPos = html_encode(job.total_positions)
 		dat += "<tr><td>[J_title]:</td> <td>[J_opPos]/[job.total_positions < 0 ? " (unlimited)" : J_totPos]"
 
-		if(job.title == "AI" || job.title == "Cyborg")

```

Comment:
Shouldn't cyborgs be left in ?

Reply:
cyborgs can latejoin properly as they'll just get spawned on the arrivals shuttle.